notificationworkerlambda/src/main/scala/com/gu/notifications/worker/delivery/fcm/FcmClient.scala

```diff
@@ -71,15 +73,28 @@ class FcmClient private (firebaseMessaging: FirebaseMessaging, firebaseApp: Fire
 
 object FcmClient {
 
+  def randomlySetOkHttpTransport(builder: FirebaseOptions.Builder): FirebaseOptions.Builder = {
+    val logger = LoggerFactory.getLogger(classOf[FcmClient])
+    val randomNumber = Random.nextInt(100)
+    val percentageOkHttpClient = 50
```

Comment:
This percentage is probably too high. What percentage should we use?

Reply:
let's start with 5 so we can see it live in the logs, then we'll ramp it up and measure if there's any impact on the NAT metrics

Reply:
Sounds good. Changed to 5